File: Rasha_Toine/Rasha/QLForm/src/qls/ast/style/Width.java
package qls.ast.style;

import ql.ast.literal.IntegerLiteral;
import qls.visiting.StyleVisitor;

public class Width extends StyleProperty {
	private IntegerLiteral widthVal;

	//constructor
	public Width(IntegerLiteral widthVal) {
		this.widthVal = widthVal;
	}

	public int getWidthVal() {
		return widthVal.getValue().getValue();
	}

	@Override
	public <T, U> T accept(StyleVisitor<T, U> visitor, U ctx) {
		return visitor.visit(this, ctx);
	}
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Width.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style
{
    public class Width : ASTNode, IStyle
    {
        public int Value { get; }

        public Width(int lineNumber, int value)
        {
            _lineNumber = lineNumber;
            Value = value;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}
